### ISSUE TITLE
InputSelect is not working when diagram is inside Antd Tabs component

### ISSUE THREAD

**Issue by DenysVuika** [bpmn-io/dmn-js#615]
__Describe the Bug__

The dropdown components in Decision Table and Literal Expression editors are not working when dmn-js is child of the Tab component.

The following component is having issues: [InputSelect](https://github.com/bpmn-io/dmn-js/blob/develop/packages/dmn-js-shared/src/components/InputSelect.js)

__Steps to Reproduce__

1. Visit the playground: https://codesandbox.io/s/dmnjs-input-bug-72267?file=/index.js
2. Select the shape and switch to the Decision Table mode
3. Try to change the Hit Policy in the dropdown

__Actual Behavior__

The dropdown is not working with the mouse, clicking any dropdown item has no effect.
Notes: the keyboard is working fine.

__Expected Behavior__

The input select component should work.

__Additional Notes__

The Sandbox includes the working scenario too. Just comment out the "tabs" example and uncomment the "Scenario 1" to see InputSelect working again.

__Environment__

Please complete the following information:

 - Browser: Microsoft Edge (Dev 89.0.731.0), Chrome (87.0.4280.88)
 - OS: macOS
 - Library version: 10.0.0

**Comment by nikku** [comment]
Could you be specific regarding your environment? Which version of MacOS, which version of Microsoft Edge do you use?

**Comment by barmac** [comment]
I can reproduce that in the CodeSandbox both on Chrome and Firefox.

**Comment by DenysVuika** [comment]
@nikku added Chrome and Edge versions, it happens in all browsers so far as the problem is with the component, not the browser imho

**Comment by barmac** [comment]
This happens also [when I use dmn-js@9](https://codesandbox.io/s/2cydg).

**Comment by barmac** [comment]
OK so I debugged this issue a bit and discovered that when you click an option, a `mousedown` event is fired with the tab pane `div` element as the event target. This triggers our global click handler which is responsible for closing the dropdown once you click outside of it.

![image](https://user-images.githubusercontent.com/28307541/103523930-fa77d280-4e7c-11eb-81e9-093380ed49f2.png)

Why is the tab pane set as the event target? It appears that it's a focusable element since it has `tabindex` property set to `0`.

![image](https://user-images.githubusercontent.com/28307541/103524377-c3ee8780-4e7d-11eb-9924-ace57b21e5ab.png)

When I removed `tabindex` attribute from the tab pane element, the `InputSelect` dropdown worked correctly again. So for now I am inclined to think that the problem is within the `ant-design` tabs as I don't see a reason why a container should be able to be focused and how it would receive user input.

There is also [a closed issue](https://github.com/ant-design/ant-design/issues/18798) in the ant-design repo which might be related.

**Comment by barmac** [comment]
tldr;

`InputSelect` handles `mousedown` events in the following way:
1) target is within the dropdown -> do nothing
2) target is outside of the dropdown -> close the dropdown

When you place `dmn-js` within an `ant-design` tab, pushing mouse button sets the whole tab as the event target even if mouse cursor is on the dropdown.

**Comment by DenysVuika** [comment]
@barmac thanks a lot for investigation!

**Comment by barmac** [comment]
I'll close this issue for now as I don't think we can do anything to fix it.

**Comment by nikku** [comment]
Given further user feedback (https://github.com/bpmn-io/dmn-js/issues/755#issuecomment-1521554200) we may want to revisit this issue. Seems we generally don't embed nicely into modals.

**Comment by gogroup-official** [comment]
I recreated it here:
https://codesandbox.io/s/material-ui-modal-demo-forked-7oim3u?file=/demo.js
